src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/DWMColorization.cs

```diff
@@ -0,0 +1,682 @@
+        );
+    }
+
+    /// <summary>
+    /// HSV representation models how colors appear under light.
+    /// </summary>
+    /// <returns><see langword="float"/> hue, <see langword="float"/> saturation, <see langword="float"/> brightness</returns>
+    private static (float Hue, float Saturation, float Value) ToHsv(Color color)
+    {
+        int red = color.R;
+        int green = color.G;
+        int blue = color.B;
+
+        var max = Math.Max(red, Math.Max(green, blue));
+        var min = Math.Min(red, Math.Min(green, blue));
+
+        var fDelta = (max - min) / _byteMax;
+
+        float hue;
+        float saturation;
+        float value;
+
+        if (max <= 0)
+        {
+            return (0f, 0f, 0f);
+        }
+
+        saturation = fDelta / (max / _byteMax);
+        value = max / _byteMax;
+
+        if (fDelta <= 0.0)
+        {
+            return (0f, saturation * 100f, value * 100f);
+        }
+
+        if (max == red)
+        {
+            hue = ((green - blue) / _byteMax) / fDelta;
+        }
+        else if (max == green)
+        {
+            hue = 2f + (((blue - red) / _byteMax) / fDelta);
+        }
+        else
+        {
+            hue = 4f + (((red - green) / _byteMax) / fDelta);
+        }
+
+        if (hue < 0)
+        {
+            hue += 360;
+        }
+
+        return (hue * 60f, saturation * 100f, value * 100f);
+    }
+
+    /// <summary>
+    /// HSL representation models the way different paints mix together to create colour in the real world,
+    /// with the lightness dimension resembling the varying amounts of black or white paint in the mixture.
+    /// </summary>
+    /// <returns><see langword="float"/> hue, <see langword="float"/> saturation, <see langword="float"/> lightness</returns>
+    private static (float Hue, float Saturation, float Lightness) ToHsl(Color color)
+    {
+        int red = color.R;
+        int green = color.G;
+        int blue = color.B;
+
+        var max = Math.Max(red, Math.Max(green, blue));
+        var min = Math.Min(red, Math.Min(green, blue));
+
+        var fDelta = (max - min) / _byteMax;
+
+        float hue;
+        float saturation;
+        float lightness;
+
+        if (max <= 0)
+        {
+            return (0f, 0f, 0f);
+        }
+
+        saturation = 0.0f;
+        lightness = ((max + min) / _byteMax) / 2.0f;
+
+        if (fDelta <= 0.0)
+        {
+            return (0f, saturation * 100f, lightness * 100f);
+        }
+
+        saturation = fDelta / (max / _byteMax);
+
+        if (max == red)
+        {
+            hue = ((green - blue) / _byteMax) / fDelta;
+        }
+        else if (max == green)
+        {
+            hue = 2f + (((blue - red) / _byteMax) / fDelta);
+        }
+        else
+        {
+            hue = 4f + (((red - green) / _byteMax) / fDelta);
+        }
+
+        if (hue < 0)
+        {
+            hue += 360;
+        }
+
+        return (hue * 60f, saturation * 100f, lightness * 100f);
```

Comment:
I do not think `hue` should add 360 in the Line 548, and I think `hue` should add `6` here. Because it will return `hue * 60f` in Line 551.

Reply:
We have removed the calculation of accent color variations and hence this is no longer in use.